extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -75,5 +72,23 @@ describes.realWin(
       expect(shoppingTag.element.textContent).to.be.empty;
       expect(shoppingTag.isLayoutSupported(Layout_Enum.CONTAINER)).to.be.true;
     });
+
+    it('should set active product in store service when shopping tag is clicked', async () => {
+      const tagData = {
+        'product-tag-id': 'sunglasses',
+        'product-title': 'Spectacular Spectacles',
+        'product-price': '400',
+        'product-icon':
+          '/examples/visual-tests/amp-story/img/shopping/nest-audio-icon.png',
+      };
+
+      await shoppingTag.onClick_(tagData);
```

Comment:
You should call `.click()` on the element instead, which will call the callback automatically (if correctly bound). We should never call private methods directly

Reply:
Yes, sounds good. I have updated it to reflect this change.